docs/source-pytorch/advanced/post-training quantization.rst

```diff
@@ -0,0 +1,158 @@
+==========================
+Post-training Quantization

```

Comment:
The page is currently not referenced anywhere. We need place it somewhere, otherwise it won't be found.

Perhaps it could be integrated here: https://pytorch-lightning.readthedocs.io/en/stable/advanced/pruning_quantization.html?highlight=quantization

Reply:
Hi, Adrian, I just update this commit and add some descriptions in index.rst and pruning_quantization.rst. I still keep the main storyline of the original introduction part but append the description about Post-training quantization with Intel Neural Compressor. Hope for your kind suggestions on my changes.

Reply:
Thank you! I will confirm with @edenlightning and then we can merge it!